company-profile-en.md

```diff
@@ -20,9 +20,9 @@
 | --- | --- | --- | --- | --- | --- |
 | Tokyo | 18 | 7 | 3 | 4 | 32 |
 | Manila | 6 | 2 | 1 | 0 | 9 |
-| Jakarta | 4 | 2 | 0 | 3 | 9 |
+| Jakarta | 7 | 4 | 0 | 3 | 9 |
```

Comment:
The total of the product team (rightmost) should be updated accordingly too, 7+4+3=14.

Reply:
@irfanfadilah right, nice catch, I didn't do any calculation before :sweat:, could you open pull request to fix it?

thanks! :dancer:

Reply:
@fudanchii added: https://github.com/quipper/handbook/pull/36 👍